avocado/utils/ssh.py

```diff
@@ -188,9 +188,17 @@ def cmd(self, command):
 
         :param command: the command to execute over the SSH session
         :type command: str
+        :param sudo: Whether the command requires admin privileges to run, so
+                     that sudo will be prepended to the command.  The
+                     assumption here is that the user running the command has a
+                     sudo configuration such that a password won't be prompted.
+                     If that's not the case, the command will straight out
+                     fail. Default is False.
         :returns: The command result object.
         :rtype: A :class:`avocado.utils.process.CmdResult` instance.
         """
+        if sudo:
+            command = "sudo {}".format(command)
```

Comment:
If the objective is to make the module fully support _sudo_, I think this is a partial solution because it does not cover the `get_raw_ssh_command` method.

Reply:
The objective was to add `sudo` only for the `cmd()` method.

I was reading for the docstring at `get_raw_ssh_command`:

> This should only be used if you need to interact with the ssh subprocess, and most users will *NOT* need to.  Try to use the :meth:`cmd` method instead.